src/ui/cherrypickAndCreateTargetPullRequest.ts

```diff
@@ -79,6 +80,11 @@ export async function cherrypickAndCreateTargetPullRequest({
     );
   }
 
+  // make PR auto mergable
+  if (options.autoMerge) {
+    await enablePullRequestAutoMerge(options, targetPullRequest.number);
```

Comment:
@brianseeders The problem was that you were using `options.pullNumber` which is the pull number of the source PR (the one that was already merged in). We need the target pull request number.

Reply:
I might rename `options.pullNumber` to `option.sourcePullNumber` to make this more clear.

Reply:
Oh my gosh, that's such a silly error on my part. Thanks for investigating.

Reply:
No worries. Thanks for landing this in the first place.